official/vision/beta/projects/panoptic_maskrcnn/configs/panoptic_maskrcnn.py

```diff
@@ -94,6 +110,16 @@ class Losses(maskrcnn.Losses):
   semantic_segmentation_weight: float = 1.0
 
 
+@dataclasses.dataclass
+class PanopticQualityEvaluator(hyperparams.Config):
+  """Panoptic Quality Evaluator config."""
+  num_categories: int = 2
+  ignored_label: int = 0
```

Comment:
ignore_label (no 'd').

Reply:
I was trying to match it with the PQ evaluator args. Should I change it here as well?
https://github.com/tensorflow/models/blob/2af8ebcd93c5449164b8f1b4d7ee2e15dd068040/official/vision/beta/evaluation/panoptic_quality_evaluator.py#L34-L40